inc/Engine/Admin/Settings/Page.php

```diff
@@ -916,7 +926,25 @@ private function assets_section() {
 					'input_attr'        => [
 						'disabled' => get_rocket_option( 'delay_js' ) ? 0 : 1,
 					],
-					'helper'            => DelayJSSettings::exclusion_list_has_default() ? $delay_js_found_list_helper : $delay_js_list_helper,
+					'items'             => $this->delayjs_sitelist->prepare_delayjs_ui_list(),
+				],
+				'delay_js_exclusions'          => [
+					'type'              => 'custom_textarea',
+					'label'             => __( 'My Scripts', 'rocket' ),
+					'description'       => __( 'Specify URLs or keywords that can identify inline or JavaScript files to be excluded from delaying execution (one per line).', 'rocket' ),
+					'container_class'   => [
+						'wpr-field--children',
+					],
+					'parent'            => 'delay_js',
+					'section'           => 'js',
+					'page'              => 'file_optimization',
+					'default'           => [],
+					'sanitize_callback' => 'sanitize_textarea',
+					'input_attr'        => [
+						'disabled' => get_rocket_option( 'delay_js' ) ? 0 : 1,
+					],
+					// 'helper'            => DelayJSSettings::exclusion_list_has_default() ? $delay_js_found_list_helper : $delay_js_list_helper,
+					'placeholder'       => __( 'ex : /wp-includes/js/jquery/jquery.min.js', 'rocket' ),
```

Comment:
```suggestion
					'placeholder'       => __( 'ex : /wp-includes/js/jquery/jquery.min.js', 'rocket' ),
```

We don't usually handle the placeholder that way